packages/vm/lib/evm/opFns.ts

```diff
@@ -893,9 +922,24 @@ function jumpIsValid(runState: RunState, dest: number): boolean {
   return runState.validJumps.indexOf(dest) !== -1
 }
 
-function maxCallGas(gasLimit: BN, gasLeft: BN): BN {
-  const gasAllowed = gasLeft.sub(gasLeft.divn(64))
-  return gasLimit.gt(gasAllowed) ? gasAllowed : gasLimit
+// returns the max call gas which we want to use for the gas limit
+// note that pre-TW there was no hard limit. if you tried sending more gas, your CALLs (or equivalent) went OOG
+//
+
+/**
+ * Returns an overflow-safe slice of an array. It right-pads
+ * the data with zeros to `length`.
+ * @param {BN} gasLimit - requested gas Limit
+ * @param {BN} gasLeft - current gas left
+ * @param {boolean} isTangerineWhistleOrLater - set to true if we are on TW or later (this converts this to the identify function if it is false)
+ */
+function maxCallGas(gasLimit: BN, gasLeft: BN, isTangerineWhistleOrLater: boolean): BN {
+  if (isTangerineWhistleOrLater) {
```

Comment:
Why not use the `Common` instance here directly?

Reply:
This makes more sense. Will update.

Reply:
Won't make this a blocker, but this constant is really a bit unnecessary. 😄